src/shared-components/container/style.ts

```diff
@@ -1,13 +1,11 @@
 import { DetailedHTMLProps, HTMLAttributes } from 'react';
 import styled, { StyledComponent } from '@emotion/styled';
-import { css } from '@emotion/core';
 import PropTypes from 'prop-types';
+import { ThemeType } from 'src/constants/themes/types';
 
-import {
- SPACER, BOX_SHADOWS, COLORS, MEDIA_QUERIES, 
-} from '../../constants';
+import { SPACER, BOX_SHADOWS, MEDIA_QUERIES } from '../../constants';
 
-const clickableStyle = css`
+const clickableStyle = `
```

Comment:
Assuming `BOX_SHADOWS` end up in here too, we'll need to change this function signature, but didn't want to move away from the focus on `COLORS` for a v1.